zerver/webhooks/statuspage/doc.md

```diff
@@ -0,0 +1,24 @@
+Get Zulip notifications for your Statuspage.io subscriptions!
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+What company needs to do:
+
+Go to the **subscribers settings** in **notifications** tab and enable notifications delivery through **WEBHOOK**.
```

Comment:
Okay so a couple of points about this one:
* Is `subscribers settings` an option on the third party website? If so, shouldn't it be capitalised like `Subscriber's Settings`? If not, then you do not need to bold the text. :)
* Change it like this: `If you're a company and you'd like to enable webhook notifications, go to the ....` and so on. :)

Reply:
Subscriber's settings are for company... I guess `Subscriber's Settings` is good...